.github/workflows/nix-config-base.json

```diff
@@ -13,11 +13,11 @@
       ]
     },
     {
-      "name": "vast-ce",
+      "name": "tenzir-ce",
       "static": true,
       "upload-package-to-github": false,
       "package-stores": [
-        "gcs:tenzir-vast/static-binary-builds"
+        "gcs:tenzir-dist-private/static-binary-builds"
```

Comment:
This does not yet exist.

Reply:
I created `tenzir-dist-private` and `tenzir-dist-public` now.